pkg/captcha/captcha.go

```diff
@@ -115,21 +156,50 @@ type responseProvider struct {
 	Success bool `json:"success"`
 }
 
+type altchaResponseProvider struct {
+	Success bool `json:"verified"`
+}
+
+type altchaVerifyPayload struct {
+	Payload string `json:"payload"`
+}
+
 // Validate Verify the captcha from provider API.
 func (c *Client) Validate(r *http.Request) (bool, error) {
 	if r.Method != http.MethodPost {
 		c.log.Debug("captcha:Validate invalid method: " + r.Method)
 		return false, nil
 	}
-	var response = r.FormValue(captcha[c.provider].key + "-response")
+	var response = r.FormValue(captcha[c.provider].key)
```

Comment:
Have you test this change on other providers ?

Reply:
previously i was only testing i could load the other widgets since I didn't think i modified the validation behavior.

It has now been tested with cloudflare turnstile, and looks good on the latest commit